portfolio/src/main/webapp/style.css

```diff
@@ -1,13 +1,15 @@
 html, body {
   height: 100vh;
   font-family: "Open Sans";
-  overflow-x: hidden;
-  overflow: hidden;
-  padding-bottom: 5%;
   word-break: break-word;
   word-wrap: break-word; 
 }
 
+a {
```

Comment:
Use a meaningful name here?

Reply:
a used for 'a href', cannot change because built in to css/html